Models/Doctor.cs

```diff
@@ -1,24 +1,37 @@
-﻿using Microsoft.AspNetCore.SignalR;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace MVCCore.Models
+namespace MVCCore
 {
     public class Doctor
     {
         [Key]
         public int DoctorId { get; set; }
         [Required]
         public string DoctorName { get; set; }
+        [Required]
         public string DoctorAddress { get; set; }
+        [Required]
         public int DoctorAge { get; set; }
+        [Required]
         public string RMPNumber { get; set; }
+        [Required]
         public string HighestQualification { get; set; }
+        [Required]
         public string Speciality { get; set; }
+        public Int32 CreatedBy { get; set; }
+        public DateTime CreateDate { get; set; }
+
+        public bool IsActive { get; set; }
 
+        [AllowNull]
+        public Int32 ModifiedBy { get; set; }
+        [AllowNull]
+        public DateTime ModifiedDate { get; set; }
```

Comment:
Can't we use nullable instead of allow the null attribute?

Reply:
Allowed null with ?